samples/ConsoleApp/ConsoleApp.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0-rc.2.21480.5" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
```

Comment:
Please update the version also in `docs/USAGE.md`. _Find `6.0.0-preview.6.21352.12`_